std_msgs/Header header
float64 simtime
mushr_mujoco_ros/BodyState[] states
